README.md

```diff
@@ -192,4 +192,4 @@ https://application-backend.com/page?error=access_denied&error_description=User+
 [tag-image]: https://img.shields.io/github/tag/smartcar/javascript-sdk.svg
 
 <!-- Please do not modify or remove this, it is used by the build process -->
-[version]: 2.2.1
+[version]: 2.3.0
```

Comment:
As part of this SDK's build process, we are supposed to run `npm run prepare-release` before each release. This will regenerate our docs and use `README.mdt` (a template file) to generate the new README (for context: we template in the version number in a few places (e.g. the SDK CDN link)).

The version number at the bottom of the README is updated when you run `prepare-release`, and this allows `build/verify-release.sh` to easily tell if prepare-release has been run for the latest version number in `package.json`. If it hasn't been updated, the build fails because that means `prepare-release` hasn't been run!